go.mod

```diff
@@ -32,6 +32,8 @@ require (
 	sigs.k8s.io/yaml v1.3.0
 )
 
+require github.com/openshift/machine-api-provider-powervs v0.0.0-20220303154846-89bb5bec120e
```

Comment:
why a separate entry here? can we include in the above block itself?

Reply:
Updated it in the latest commit